x-pack/solutions/security/plugins/security_solution/public/onboarding/components/__mocks__/mocks.ts

```diff
@@ -4,23 +4,19 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
 
 export const mockReportCardOpen = jest.fn();
 export const mockReportCardComplete = jest.fn();
 export const mockReportCardLinkClicked = jest.fn();
 export const mockReportCardSelectorClicked = jest.fn();
+export const mockTrackLinkClick = jest.fn();
 
 export const telemetry = {
   reportCardOpen: mockReportCardOpen,
   reportCardComplete: mockReportCardComplete,
   reportCardLinkClicked: mockReportCardLinkClicked,
   reportCardSelectorClicked: mockReportCardSelectorClicked,
+  trackLinkClick: mockTrackLinkClick,
```

Comment:
Why do we need this in the onboarding telemtry mock? Hasn't it been moved to common/integrations?

Reply:
As this is eventually calling `trackOnboardingLinkClick` in x-pack/solutions/security/plugins/security_solution/public/onboarding/components/lib/telemetry.ts

I'm only making it available in `trackLinkClick` x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_telemetry.ts and `OnboardingContext`

so other places like `asset_inventory` wouldn't have this method and their clicks wouldn't be taken into account: x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/onboarding/no_data_found.tsx
It's currently used in the onboarding scope. What do you think?